backend/app/methods/adminMethods.js

```diff
@@ -0,0 +1,47 @@
+const { AdminUser } = require('../models')
+const { logMessage } = require('../utilities/logHelper')
+
+exports.newAdmin = async function(req, res) {
+
+  // only callparty user can create admin
+  const bot = req.user.bot
+  if (['callparty', 'callingteststaging'].indexOf(bot) < 0) {
+    throw new Error('++ only callparty users can create admin')
+  }
+
+  // parse data
+  const data = req.body
+  await logMessage(`++ @here creating new admin: ${JSON.stringify(data)}`)
+
+  // check if admin with this username or bot already exists
+  const alreadyUsername = await AdminUser.findOne({ username: data.username })
+  if (alreadyUsername) {
+    throw new Error(`++ cannot create two adminUser with username ${data.username}`)
+  }
+  const alreadyBot = await AdminUser.findOne({ bot: data.bot })
+  if (alreadyBot) {
```

Comment:
currently we actually store bot in the token (as opposed to userId)
so it would really mess things up if there were multiple admin users for the same bot

I think that could be re-visited. probably we should store a userId in the token